cli/server_test.go

```diff
@@ -210,7 +219,16 @@ func TestServer(t *testing.T) {
 		require.NotEmpty(t, rootDir)
 		require.DirExists(t, rootDir)
 
-		pty.ExpectMatchContext(ctx, "View the Web UI")
+		matchCh2 := make(chan string, 1)
+		go func() {
+			matchCh2 <- pty.ExpectMatchContext(ctx, "View the Web UI")
+		}()
+		select {
+		case err := <-errCh:
+			require.NoError(t, err)
+		case <-matchCh2:
+			// OK!
+		}
```

Comment:
Why do we actually care about this message being logged?

Reply:
It's a decent indicator that the server was successfully set up. Let me add a comment about it.